.gitlab-ci.yml

```diff
@@ -71,11 +71,10 @@ nightly-test:
   <<:                              *docker-env
   stage:                           test
   variables:
-    EXTRA_FEATURES:                "$EXTRA_FEATURES unstable"
+    EXTRA_FEATURES:                "unstable"
   script:
-    - cargo build --verbose --features="all ${EXTRA_FEATURES}" || exit 1
-    - RUST_BACKTRACE=1 cargo test --workspace --verbose --no-default-features --features="${EXTRA_FEATURES}"
-    - RUST_BACKTRACE=1 cargo test --workspace --verbose --features="all ${EXTRA_FEATURES}"
+    - cargo +nightly build --verbose --features="${EXTRA_FEATURES}" || exit 1
```

Comment:
nightly is the default, not necessary to `+nightly`

Reply:
I prefer t be explicit rather than implicit.